### PR TITLE
moved `ctrl` to `Salesforce/ctrl`

### PR DESCRIPTION
**Pull request by julien-c**
redirects should theoretically work, but still updating those repo references for clarity

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
LGTM

**Comment by LysandreJik** [comment]
Taking a look at and fixing the failures

File: utils/check_config_docstrings.py [hunk 1]
```diff
@@ -73,7 +74,9 @@ def get_checkpoint_from_config_class(config_class):
 def check_config_docstrings_have_checkpoints():
     configs_without_checkpoint = []
 
-    for config_class in list(CONFIG_MAPPING.values()):
+    a = [CTRLConfig]
```
Comment (sanchit-gandhi) [inline utils/check_config_docstrings.py:77]: Looks like this debugging statement made it into the final PR! Opened a PR to revert these changes:
